scripts/dockerbot/check_prs.py

```diff
@@ -281,6 +292,15 @@ def check_status(pr):
         pass
 
 
+def get_pr_metadata(pr_body):
```

Comment:
Is it worth extracting the guts of this to a new function that both can use? putting it in dockerbot module and importing it into the check_prs?